js/src/mesh.ts

```diff
@@ -156,6 +161,14 @@ class MeshView extends widgets.WidgetView {
         });
     }
 
+    update_shadow() {
+        // TODO: we don't have this yet in scatter (inconsistent)
```

Comment:
would be great it you can make the update_shadow code common for scatter and mesh (scatter needs this, and it needs to be called from several places)